utils/swaggerDefinition.js

```diff
@@ -53,6 +53,43 @@ const swaggerOptions = {
             }
           }
         },
+        tasks: {
+          type: 'object',
+          properties: {
+            type: {
+              type: 'string'
+            },
+            links: {
+              type: 'array',
+              items: []
```

Comment:
Can you define the data to be required in the items array?

Reply:
Like this `items: [link1, link2]` ?

Reply:
Also data types for those.
Refer:
- https://swagger.io/docs/specification/2-0/adding-examples/
- https://stackoverflow.com/questions/35955243/following-swagger-specifications-how-can-i-define-json-of-nested-objects-to-yam

Reply:
Ok Thanks.